### PR TITLE
Pre-version is ready

### PR DESCRIPTION
**Pull request by EgorTurets**
Not finished text blocks in info lines and bottom of the page.

### CODE REVIEW

File: P1/.idea/inspectionProfiles/Project_Default.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<component name="InspectionProjectProfileManager">
```
Comment (alexanderturuta) [inline P1/.idea/inspectionProfiles/Project_Default.xml:1]: you don't need to commit your workspace configuration to the repository. please add all necessary files to gitignore.

File: P1/index.html [hunk 2]
```diff
@@ -1,14 +1,134 @@
...
 </head>
 <body>
-<div>12345</div>
-<i class="fa fa-pencil fa-2x" aria-hidden="true"></i>
-<i class="fa fa-users fa-2x" aria-hidden="true"></i>
-<i class="fa fa-wrench fa-2x" aria-hidden="true"></i>
+    <div class="background-div" align="center">
+        <div class="logo">SITENAME</div>
+
+        <div align="center">
```
Comment (alexanderturuta) [inline P1/index.html:15]: It's a good practice to use semantic elements, such like header, section. div tells nothing about what this part of markup means.

File: P1/index.html [hunk 3]
```diff
@@ -1,14 +1,134 @@
...
+            <a href="#" class="menu-item active">HOME</a>
+            <a href="#" class="menu-item">ABOUT</a>
+            <a href="#" class="menu-item">SERICES</a>
```
Comment (alexanderturuta) [inline P1/index.html:18]: looks like a typing error -> `SERICES`

File: P1/index.html [hunk 4]
```diff
@@ -1,14 +1,134 @@
...
+    <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
     <meta charset="UTF-8">
     <title>Title</title>
 </head>
 <body>
-<div>12345</div>
-<i class="fa fa-pencil fa-2x" aria-hidden="true"></i>
-<i class="fa fa-users fa-2x" aria-hidden="true"></i>
-<i class="fa fa-wrench fa-2x" aria-hidden="true"></i>
+    <div class="background-div" align="center">
```
Comment (alexanderturuta) [inline P1/index.html:12]: `align` for div is obsolete in html5. use css instead.

File: P1/index.html (hunk 2 above)
Comment (alexanderturuta) [inline P1/index.html:15]: `align'

File: P1/index.html [hunk 5]
```diff
@@ -1,14 +1,134 @@
...
+            <button class="red-button">READ MORE</button>
+        </div>
+
+        <div>
+            <h1>VIVAMUS VEST</h1>
+            <p class="main-text">Sed adipiscing ornare risus. Morbi est est, blandit sit amet, sagittis vel, euismod vel, velit.<br>
+                Pellentesque egestas sem. Suspedisse commodo ullamcorper magna.</p>
+        </div>
+
+
```
Comment (alexanderturuta) [inline P1/index.html:45]: too much new lines.

File: P1/index.html [hunk 6]
```diff
@@ -1,14 +1,134 @@
...
+        <div class="info-line bottom-border">
+            <div class="info-box1">
+                <div class="icon-background">
+                    <i class="fa fa-pencil icon" aria-hidden="true"></i>
+                </div>
+
+                <p class="bold-text-info">FUSCE PELLENTESQUE SUSCIPIT NIBH</p>
+                <p class="info-text">Sed egestas, ante et vulputate voultpat, eros pede semper est, vitae luctus metus libero eu augue. Morbi purus libero, id...</p>
+                <button class="bold-text-info more-info-button">MORE INFO&nbsp<i class="fa fa-chevron-right" aria-hidden="true" style="color: #ed3237"></i></button>
```
Comment (alexanderturuta) [inline P1/index.html:56]: `&nbsp;` is the correct way

Comment (alexanderturuta) [inline P1/index.html:56]: do not use inline styles.

File: P1/index.html [hunk 7]
```diff
@@ -1,14 +1,134 @@
...
+                    <i class="fa fa-wrench fa-2x icon" aria-hidden="true"></i>
+                </div>
+                <p class="bold-text-info">VESTIBULUM COMMODO FELIS QUTIS TORT</p>
+                <p class="info-text">Tegestas, ante et vulputate volutpat, eros pede semper est, vitae luctus metus libero eu augue. Morbi purus libero, fauc...</p>
+                <button class="bold-text-info more-info-button">MORE INFO&nbsp<i class="fa fa-chevron-right" aria-hidden="true" style="color: #ed3237"></i></button>
+            </div>
+        </div>
+
+        <!--<hr/>-->
+        <div>
```
Comment (alexanderturuta) [inline P1/index.html:77]: what is the purpose to wrap into the empty `div`?

File: P1/index.html [hunk 8]
```diff
@@ -1,14 +1,134 @@
-<!DOCTYPE html>
+﻿<!DOCTYPE html>
 <html lang="en">
 <head>
     <link rel="stylesheet" href="font-awesome-4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="InfoBoxes.css">
```
Comment (alexanderturuta) [inline P1/index.html:6]: good thing to follow one style of file naming convention.

File: P1/style.css [hunk 9]
```diff
@@ -0,0 +1,176 @@
...
+    text-align: center;
+    margin-bottom: 50px;
+}
+
+div.read-more {
+    padding: 0px;
+    width: 155px;
+    position: relative;
+    margin-top: -31px;
+    padding-bottom: 20px;
```
Comment (alexanderturuta) [inline P1/style.css:42]: overrides padding above from the same class `padding: 0px;`. can be written in one line

File: P1/style.css [hunk 10]
```diff
@@ -0,0 +1,176 @@
...
+    background: linear-gradient(90deg, rgb(44,44,44) 50%, rgb(50,50,50) 50%,rgb(50,50,50) 55%, rgb(40,40,40) 55%);
+    background-size: 40px 40px;
+    background-color: black;
+    padding-top: 39px;
+    font-family: "Source Sans Pro";
+    /*letter-spacing: 1px;*/
+}
+
+div {
+    padding-top: 21px;
```
Comment (alexanderturuta) [inline P1/style.css:11]: bad practice to use a common style for `div` in such way. padding all the things :)

File: P1/style.css [hunk 11]
```diff
@@ -0,0 +1,176 @@
...
+    margin: -65px auto 21px auto;
+    padding: 25px;
+    border-radius: 5px;
+    background-color: #ebebeb;
+    box-shadow: -10px 10px rgba(0, 0, 0, 0.1);
+    text-align: center;
+    font-size: 40px;
+    font-weight: bold;
+    letter-spacing: 3px;
+    z-index: 9999;
```
Comment (alexanderturuta) [inline P1/style.css:83]: is z-index necessary?

File: P1/style.css [hunk 12]
```diff
@@ -0,0 +1,176 @@
...
+}
+
+.red-button {
+    /*height: 40px;*/
+    width: 118px;
+    /*padding: 5px 20px 5px 20px;*/
+    border: 0px;
+    border-radius: 5px;
+    box-shadow: -4px 4px rgba(0, 0, 0, 0.1);
+    /*font-size: 13px;*/
```
Comment (alexanderturuta) [inline P1/style.css:131]: remove the comments

Comment (alexanderturuta) [reply P1/style.css:131]: unnecessary I mean

File: P1/style.css [hunk 13]
```diff
@@ -0,0 +1,176 @@
...
+    background-color: #ebebeb;
+    box-shadow: -10px 10px rgba(0, 0, 0, 0.1);
+    text-align: center;
+    font-size: 40px;
+    font-weight: bold;
+    letter-spacing: 3px;
+    z-index: 9999;
+}
+
+.background-div {
```
Comment (alexanderturuta) [inline P1/style.css:86]: use more meaningful naming. `background-div` is not a good choice. So couldn't you use it for `<section>`?

File: P1/style.css [hunk 14]
```diff
@@ -0,0 +1,176 @@
...
+    padding: 5px;
+    padding-left: 20px;
+    padding-right: 20px;
+    box-shadow: -5px 5px rgba(0, 0, 0, 0.1);
+    /*font-size: 13px;*/
+    font-weight: bold;
+    text-align: center;
+}
+
+.add-button-margin {
```
Comment (alexanderturuta) [inline P1/style.css:153]: what can I do with this class?

File: P1/style.css [hunk 15]
```diff
@@ -0,0 +1,176 @@
...
+
+.transparent-button {
+    background-color: #ebebeb;
+    border: 1px solid #cecece;
+    border-radius: 5px;
+    /*height: 40px;*/
+    margin: 5px;
+    padding: 5px;
+    padding-left: 20px;
+    padding-right: 20px;
```
Comment (alexanderturuta) [inline P1/style.css:146]: padding can be rewritten in one line

File: P1/style.css [hunk 16]
```diff
@@ -0,0 +1,176 @@
...
+    box-shadow: -4px 4px rgba(0, 0, 0, 0.1);
+    /*font-size: 13px;*/
+    font-weight: bold;
+    background-color: #ed3237;
+    color: #ebebeb;
+    text-align: center;
+}
+
+.transparent-button {
+    background-color: #ebebeb;
```
Comment (alexanderturuta) [inline P1/style.css:139]: `transparent-button` has actually a background color...

File: P1/index.html [hunk 17]
```diff
@@ -1,14 +1,134 @@
-<!DOCTYPE html>
```
Comment (alexanderturuta) [inline P1/index.html:1]: unnoticable on the current background but still exists
image

File: P1/index.html (hunk 2 above)
Comment (alexanderturuta) [inline P1/index.html:15]: think about `<nav>` https://developer.mozilla.org/en/docs/Web/HTML/Element/nav

File: P1/index.html [hunk 18]
```diff
@@ -1,14 +1,134 @@
...
+        <hr class="double-line"/>
+        <div class="read-more">
+            <button class="red-button">READ MORE</button>
+        </div>
+
+        <div>
+            <h1>VIVAMUS VEST</h1>
+            <p class="main-text">Sed adipiscing ornare risus. Morbi est est, blandit sit amet, sagittis vel, euismod vel, velit.<br>
+                Pellentesque egestas sem. Suspedisse commodo ullamcorper magna.</p>
+        </div>
```
Comment (alexanderturuta) [inline P1/index.html:43]: empty `<div>` wrappings

File: P1/index.html [hunk 19]
```diff
@@ -1,14 +1,134 @@
...
+        <div>
+            <img class="main-image" src="images/main_img.jpg">
+        </div>
+
+        <div>
+            <h1>LOREM IPSUM</h1>
+            <p class="main-text">Donec nec justo eget felis facilisis fermentum.<br>
+                Aliquam porttit or mauris sit amet orci. Aenean dignissim pellentesque felis.</p>
+        </div>
+        <div class="bottom-border"></div>
```
Comment (alexanderturuta) [inline P1/index.html:32]: empty div just for padding. you can style div above

File: P1/index.html [hunk 20]
```diff
@@ -1,14 +1,134 @@
...
+        </div>
+
+        <!--<hr/>-->
+        <div>
+            <h1>CRAS IACULIS</h1>
+            <p class="main-text">Fusce lobortis lorem at ipsum semper sgittis</p>
+        </div>
+
+        <div class="info-line bottom-border">
+            <div class="info-box2">
```
Comment (alexanderturuta) [inline P1/index.html:83]: info-box1, 2, 3 ....N. You need to better organize your styles.

File: P1/index.html [hunk 21]
```diff
@@ -1,14 +1,134 @@
...
+        </div>
+
+        <hr class="double-line"/>
+        <div class="read-more">
+            <button class="red-button">VIEW MORE</button>
+        </div>
+    </div>
 </body>
+<footer>
+    Copyright(c) website name.<br>
```
Comment (alexanderturuta) [inline P1/index.html:131]: footer out of body is not valid

File: P1/style.css [hunk 22]
```diff
@@ -0,0 +1,176 @@
+BODY {
```
Comment (alexanderturuta) [inline P1/style.css:1]: just tips for the future. take a look at some best practices. for example http://cssreset.com/what-is-a-css-reset/

File: P1/style.css [hunk 23]
```diff
@@ -0,0 +1,176 @@
...
+    background: linear-gradient(90deg, rgb(44,44,44) 50%, rgb(50,50,50) 50%,rgb(50,50,50) 55%, rgb(40,40,40) 55%);
+    background-size: 40px 40px;
+    background-color: black;
+    padding-top: 39px;
+    font-family: "Source Sans Pro";
```
Comment (alexanderturuta) [inline P1/style.css:6]: default font is missed

**Review by alexanderturuta — COMMENTED** [review]
see comments

File: P1/style.css (hunk 23 above)
Comment (EgorTurets) [reply P1/style.css:6]: don't understand. I used a font, as in the example.

File: P1/index.html (hunk 3 above)
Comment (EgorTurets) [reply P1/index.html:18]: like in a example.

**Comment by Turach** [comment]
Looks ok, but need to cover next items:
* Above comments from Alexander
* Ensure page looks the same in Chrome, IE, FF (latest versions)
* Cursor should be pointer on all button/links
* All buttons should have color inversion on hover
* Top navigation links should have hover color as current active one
* Need to make basic "responsiveness" for page blocks and header navigation

**Pull request merged**